### ISSUE TITLE
[Accessibility] High contrast theme

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2407]
- [x] All text is white
- [x] Background is black 
- [x] Selected / Highlight is light blue
- [x] Disabled is light green
- [x] Buttons: black background, color white, border white (solid 1px white)
- [x] Structure should be outlined with borders (simulator, editor toolbar, main menu, blockly / monaco toolbox)
- [x] Blockly / monaco toolbox with no color, white for text, black for background.
- [ ] Outline menu items

I did some investigation myself, so have a look at the accessibility-highcontrast branch as a start into high contrast work: 
<img width="1304" alt="screen shot 2017-07-06 at 8 11 40 pm" src="https://user-images.githubusercontent.com/16690124/27923530-60ec4a5e-6287-11e7-9963-cec760b000a1.png">
